python/cucim/requirements-test.txt

```diff
@@ -1,5 +1,6 @@
 GPUtil>=1.4.0
 imagecodecs>=2021.6.8
+numpydoc>=1.5
```

Comment:
Also think we can fold this into `dependencies.yaml`. Trying that in PR ( https://github.com/rapidsai/cucim/pull/683 )

Reply:
This file was folded into `pyproject.toml`

So would also suggest adding this to `dependencies.yaml`